### ISSUE TITLE
Feature request: Unigram tokenizer

### ISSUE THREAD

**Issue by amakropoulos** [huggingface/sharp-transformers#8]
Unigram tokenizers are used in the best multilingual models e.g. multilingual-e5, paraphrase-multilingual-MiniLM-L12-v2.
It would be very helpful if you could add an implementation to expand the functionality to different languages.

**Comment by simoninithomas** [comment]
Agreed. We're going to add more tokenizers features (LLM, Audio, TTS etc)  starting beginning of April

**Comment by amakropoulos** [comment]
Perfect, enjoy GDC 🙂!
